### PR TITLE
Proverif 1.91

### PR DESCRIPTION
**Pull request by beurdouche**

### CODE REVIEW

**Comment by dpo** [comment]
```
==> brew audit homebrew/science/proverif
==> FAILED
Error: 1 problem in 1 formula
homebrew/science/proverif:
 * Emacs Lisp files were linked directly to /usr/local/share/emacs/site-lisp

This may cause conflicts with other packages; install to a subdirectory instead, such as
/usr/local/Cellar/proverif/1.91/share/emacs/site-lisp/proverif

The offending files are:
  /usr/local/Cellar/proverif/1.91/share/emacs/site-lisp/proverif.el
```
I suspect changing `share` to `pkgshare` on line 22 would resolve the issue.

**Comment by dpo** [comment]
@BrewTestBot test this please

**Comment by beurdouche** [comment]
@dpo :
`brew audit proverif.rb --strict --online`
`brew install --verbose --debug proverif.rb`

This is strange, I am up to date and both commands indicate the formula is fine...

File: proverif.rb [hunk 1]
```diff
@@ -19,14 +20,10 @@ def install
     doc.install Dir["docs/*"], "README", "examples"
 
     (prefix/"cryptoverif").install "cryptoverif.pvl"
-    (share/"emacs/site-lisp").install Dir["emacs/*"]
+    (pkgshare/"emacs/site-lisp").install Dir["emacs/*"]
```
Comment (xu-cheng) [inline proverif.rb]: I think this should be `(share/"emacs/site-lisp/proverif").install Dir["emacs/*"]`

**Comment by xu-cheng** [comment]
> This is strange, I am up to date and both commands indicate the formula is fine...

If you run `brew audit` after `brew install`, it will do more check on the installation files.

**Comment by beurdouche** [comment]
@dpo @xu-cheng : I just double checked by doing `brew audit` after a clean `brew install`... no changes.
I'll try to change the emacs line as you suggested...

**Comment by beurdouche** [comment]
@dpo, @xu-cheng : no change, everything passes locally, no clue why merging check is failing =s

**Comment by beurdouche** [comment]
@dpo, @xu-cheng : \o/
Thank you both for the help !

File: proverif.rb (hunk 1 above)
Comment (dpo) [inline proverif.rb]: Oops. Sorry, thanks.

**Pull request closed without merging**